scan.py

```diff
@@ -266,10 +268,7 @@ def start(self):
 
         for username, account in self.accounts.items():
             account['username'] = username
-            if account.get('captcha'):
-                self.captcha_queue.put(account)
```

Comment:
I'll change the handling, the `captcha_queue`, which will now only contain accounts which have had more than `CAPTCHAS_ALLOWED` CAPTCHAs, and they will then have to be deCAPTCHAed manually
Otherwise handling will be as it has been before